src/core/ItemContainer/index.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const ItemContainer = styled.li`
+  border: 1px solid;
+  border-radius: 5px;
+  border-color: #dad9d9;
+  background-color: #f4f4f4;
+  list-style: none;
+  margin-top: 10px;
+  padding: 5px 10px;
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  align-items: center;
+  letter-spacing: 1px;
+  font-size: large;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  max-width: 250px;
+  min-width: 250px;
```

Comment:
```suggestion
  width: 250px;
```

Since min and max props have the same value, they can be condensed in `width`, it is the same